CallsheetLocalizations/de.xcloc/Localized Contents/de.xliff

```diff
@@ -766,7 +766,7 @@ Sie enthält derzeit %2$lld pins.</target>
       </trans-unit>
       <trans-unit id="Hand your phone to someone knowing they **cannot mess up your photos**." xml:space="preserve">
         <source>Hand your phone to someone knowing they **cannot mess up your photos**.</source>
-        <target state="translated">**Deine Fotos sind in sicheren Händen**, selbst wenn dein Handy grad nicht in deinen ist.</target>
+        <target state="translated">**Deine Fotos sind in sicheren Händen**, selbst wenn dein Handy gerade nicht in deinen ist.</target>
```

Comment:
"grad" is kind of slangy, "gerade" is the correct word → more professional

Reply:
I trust your judgement, but given "mess up" is also pretty casual, does that change your opinion?

Reply:
Nope, because the German translation here is not word-for-word. Word-for-word it says:
"Your photos are in safe hands, even when your phone isn't at the moment" (meaning that the hands your phone is in might not be safe, because they might be sniffing around).

If chose to only adjust the "slang" here, but if you want, I can change the string to something that resembles the English version more.

Reply:
Ah, gotcha! 👍 Again, I trust your ear/judgement. If you think this is fine (after your correction), I'm good with that. If you think you can take a different cut at it that sounds better, that's fine too. Either way, once you decide, resolve this conversation and I'll merge this PR! :tada:

Reply:
Every alternative translation I come up with sounds weird 😅 IMO one cannot express the same thing in German as concise and as elegant as in English. That's why I think I'd leave it like that for now.

Reply:
👍🏻